_i18n/nb-no/resources/user-guides/multisig-messaging-system.md

```diff
@@ -0,0 +1,864 @@
+
+Hvis en annen signerer forteller deg at en bestemt melding ikke ankom eller gikk tapt, kan du når som helst sende den igjen ved å plukke den opp fra meldingslisten slik du vanligvis hadde sendt en e-post på nytt i en liknende situasjon.
+
+### Signerere og meldinger
+
+Så der en «normal» Monero-lommebok uten MMS kun håndterer tre typer data (adresser, kontoer og transaksjoner), legger MMS-en til to til: Signerere og meldinger.
+
+MMS-en håndterer, for hver separate multisig-lommebok, en liste over *autoriserte signerere*. Med 2/3 multisig, består den listen av **tre** oppføringer. På et teknisk nivå representerer hver oppføring en Monero-lommebok som inneholder nøkler som kan brukes til å signere multisig-transaksjoner. På et konseptuelt nivå er det lettere å forestille seg en gruppe med tre personer, f.eks. deg selv og to partnere, som de «autoriserte signererne». (Det vil ofte være spesielt tre personer som kontrollerer de tre lommebøkene, men ikke alltid, selvfølgelig.)
+
+MMS-en håndterer også en enkel liste med *meldinger* per lommebok: Alle meldingene du sender pluss alle meldingene du motta. Mens listen over autoriserte signerere er den samme i alle de involverte lommebøkene, skiller meldingene seg sevfølgelig fra hverandre. Jo flere autoriserte signerere det er som kan sende deg en melding, og jo lengre du foretar transaksjoner, jo flere meldinger vil samle seg opp.
+
+## Å få tak i MMS-en
+
+Per dags dato (slutten av 2018), er MMS-en kun tilgjengelig som del av den siste Monero-koden (`master`-grenen i Moneros [GitHub-arkiv](https://github.com/monero-project/monero)). For å bruke det, må du kontrollere kildekoden og kompilere den på egenhånd. Det er lettest å gjøre dette på et Linux-system.
+
+Med den neste hardforken som kom våren 2019, ble MMS-en en integrert, standard del av Monero-programvaren: Du har det når du installerer Monero.
+
+Men en liten advarsel: Å bruke den siste Monero-versjonen fører per dags dato ikke til konflikter og komplikasjoner med vanlig utgitt Monero-programvare og nedlastet blokkjede på samme system, men det kan ha endret seg da hardforken fant sted.
+
+## Installasjon og konfigurering av PyBitmessage
+
+Å installere PyBitmessage er lett nok: Du finner lenker til å laste ned og installeringsinstrukser på [Bitmessage Wiki-hjemmesiden](https://bitmessage.org/wiki/Main_Page). Det finnes versjoner for alle de store operativsystemene som Monero også støtter: Linux, Windows, og macOS.
+
+Etter du har installert programmet, installer og konfigurer en Bitmessage-adresse for deg selv og noter den ned, fordi du vil senere trenge den for å konfigurere multisig-lommeboken din.
+
+Ikke bekymre deg om det virker som PyBitmessage ikke kobler seg til Bitmessage-nettverket når du kjører det for føste gang. Grunnet den desentraliserte naturen til det nettverket, kan det ta litt tid før du kan koble til. Det virker som dette ofte tar en **halvtime**.

```

Comment:
Maybe change to: Ikke bekymre deg hvis det virker som PyBitmessage ikke kobler seg til Bitmessage-nettverket når du kjører det for første gang.

Reply:
fixed